docs/api/composables.md

```diff
@@ -136,12 +136,12 @@ Then you can bind the textures to the material.
       <TresSphereGeometry />
       <TresMeshStandardMaterial
         :map="map"
-        :displacementMap="displacementMap"
-        :normalMap="normalMap"
-        :roughnessMap="roughnessMap"
-        :metalnessMap="metalnessMap"
-        :aoMap="aoMap"
-        :alphaMap="alphaMap"
+        :displacement-map="displacementMap"
```

Comment:
This will break the property?

Reply:
I don't think so. It is not a nested property (https://threejs.org/docs/index.html#api/en/materials/MeshStandardMaterial.displacementMap).